### DISCUSSION TITLE
مهمم

### DISCUSSION THREAD

**Discussion by shayan041ir** [hezarai/hezar#176] [Q&A]
سلام خسته نباشید 
ببخشید من چجوری میتوانم با استفاده از هزار برنامه ای را پیاده سازی بکنم

**Comment by arxyzan** [comment]
@shayan041ir 
سلام چه برنامه ای؟

**Reply by shayan041ir** [reply]
میخوام در یک قسمت از برنامه که کارش تبدیل عکس به متن هست استفاده کنم ولی ارور dll و torch میده هی

**Reply by arxyzan** [reply]
احتمالا مشکل از ورژن پایتونت هست. با پایتون ۳.۱۱ امتحان کن.

**Reply by shayan041ir** [reply]
حل شد ❤️
